tortoise/tortoise_test.go

```diff
@@ -1698,11 +1730,13 @@ func TestVoteAgainstSupportedByBaseBallot(t *testing.T) {
 
 	unsupported := map[types.BlockID]struct{}{}
 	for lid := genesis; lid.Before(last); lid = lid.Add(1) {
-		bids, err := s.GetState(0).MeshDB.LayerBlockIds(lid)
+		theOne, err := s.GetState(0).MeshDB.GetHareConsensusOutput(lid)
 		require.NoError(t, err)
-		tortoise.trtl.validity[bids[0]] = against
-		tortoise.trtl.hareOutput[bids[0]] = against
-		unsupported[bids[0]] = struct{}{}
+		if theOne != types.EmptyBlockID {
```

Comment:
this conditiona should be removed

Reply:
this is due to the explicit `sim.WithEmptyHareOutput()`. so one of the valid block will be types.EmptyBlockID which won't be in the support list.